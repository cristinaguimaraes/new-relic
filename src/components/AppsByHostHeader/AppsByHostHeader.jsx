import React from "react";
import { func, oneOf } from "prop-types";
import { Title, Wrapper, Icon, Text } from "./styled";

const AppsByHostHeader = ({ onClick, layout }) => (
  <Wrapper>
    <Title>Apps by Host</Title>
    <Text>for user averylongemailaddress@companyname.com</Text>
    <Icon className={"material-icons"} onClick={onClick}>
      check_box_outline_blank
    </Icon>
    <Text>Show as {layout}</Text>
  </Wrapper>
);

AppsByHostHeader.propTypes = {
  onClick: func.isRequired,
  layout: oneOf(["list", "grid"]).isRequired
};

export { AppsByHostHeader };
