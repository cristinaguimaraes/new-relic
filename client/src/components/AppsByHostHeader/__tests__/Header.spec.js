import { AppsByHostHeader} from '../AppsByHostHeader';
import { shallow } from 'enzyme';
import React from 'react';

describe('AppsByHostHeader2 component', () => {
  test('should render properly', () => {
    expect(shallow(<AppsByHostHeader layout={'list'} onClick={jest.fn()}>Test</AppsByHostHeader>)).toMatchSnapshot();
  });
});
