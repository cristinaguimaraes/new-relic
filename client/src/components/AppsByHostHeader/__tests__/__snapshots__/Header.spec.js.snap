// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppsByHostHeader component should render properly 1`] = `ShallowWrapper {}`;
